package.json

```diff
@@ -0,0 +1,49 @@
+{
+  "name": "verapdf-webapp-gui-fork",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "@material-ui/core": "^4.9.4",
```

Comment:
Let's not add dependencies only for "future" use. Instead let's add dependency as soon as we actually needed.
E.g. first add "react-redux" and start using it in one commit. In next you can add another dependency, Material UI, and design a base layout using it. Etc.
This way we will avoid dependencies which are added accidentally and are not actually used. Also it will be easier to identify what for this dependency was added.

Reply:
Removed unnecessary dependencies